src/components/Contacts/ContactListTable.jsx

```diff
@@ -62,6 +72,15 @@ const ContactListTable = ({ contacts, deleteContact }) => {
       headerAlign: 'center',
       align: 'center'
     },
+    {
+      field: 'Message',
+      renderCell: (contactId) => <SendIcon onClick={() => handleSendMessage(contactId)} />,
+      sortable: false,
+      filterable: false,
+      width: 80,
```

Comment:
Think we could have all the columns with interactive buttons be the same width. So perhaps we could make the other two columns 80px width as well.